composer.json

```diff
@@ -12,6 +12,7 @@
     "ext-intl": "*",
     "ext-json": "*",
     "ext-mbstring": "*",
+    "ext-simplexml": "*",
```

Comment:
Do you mean, by adding this into composer requirements, that PrestaShop _need_ SimpleXml to function?
If yes, we should complete the devdocs ? https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/

Reply:
PrestaShop need SimpleXml : https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/controllers/http/system.php#L103

Reply:
I update the devdocs.

Reply:
https://github.com/PrestaShop/docs/pull/831 (done)